### DISCUSSION TITLE
How does servicing of MAUI app works?

### DISCUSSION THREAD

**Discussion by MagicAndre1981** [dotnet/maui#6730] [Q&A]
How does updating of MAUI framework libs to a newer version works? In [XF I can update the nuget package](https://www.nuget.org/packages/Xamarin.Forms/), compile and test if my apps still work. If I get issues I can undo the change without committing the changed csprojs.  How does MAUI framework get updated? Only via new VS2022 update? What if I get a MAUI bug that blocks me? Restore my Windows to older restore point with older VS2022 and older MAUI version? And how do I know which MAUI version I compile my app against? 

@StephaneDelcroix @davidortinau @PureWeen can anyone give an answer to this?

**Comment by Xyncgas** [comment]
It's like any native app... Either you update the app or you update the content

**Reply by Xyncgas** [reply]
through windows store updates, your own channel, apple store etc...

**Reply by MagicAndre1981** [reply]
I mean how to update the MAUI framework libs to a newer version before releasing an new app update.
